exercises/change/src/main/kotlin/ChangeCalculator.kt

```diff
@@ -0,0 +1,7 @@
+class ChangeCalculator {
```

Comment:
Shouldn't the class contain an argument like `list: List<Int>`? It wont compile otherwise

Reply:
It contains `implement proper constructor` TODO-instruction instead. 
It's ok for a stub to not fit the signature requirements dictated by tests. It can be even uncompilable. And there are cases, when a correct signature is actually the solution.

Reply:
Ok cool